src/main/java/games/strategy/net/IServerMessenger.java

```diff
@@ -35,23 +35,23 @@
    */
   Set<INode> getNodes();
 
-  void NotifyIPMiniBanningOfPlayer(String ip, Date expires);
+  void notifyIPMiniBanningOfPlayer(String ip, Instant expires);
```

Comment:
Since you already renamed this method, can you please change `IP` to `Ip` to remove the remaining Checkstyle warning on this line?